src/common/types/store-data/visualization-scan-result-data.ts

```diff
@@ -17,7 +17,13 @@ export interface TabbedElementData extends TabStopEvent {
     propertyBag?: TabOrderPropertyBag;
 }
 
-export type TabStopRequirementStatus = 'pass' | 'fail' | 'unknown';
+export enum TabStopRequirementStatuses {
```

Comment:
I thought it might be good for us to be able to reference the statuses directly instead of hardcoding their strings places